olegbetsun/booking_up_for_beauty.go

```diff
@@ -0,0 +1,52 @@
+package booking
+
+import (
+	"time"
+	"strconv"
+	"fmt"
+	)
+
+func ParseDate(layout string, date string) time.Time{
+	t,err := time.Parse(layout, date)
+	if err != nil {
+		fmt.Printf("%v \n", err)
+	}
+	return t
+}
+
+
+// Schedule returns a time.Time from a string containing a date
+func Schedule(date string) time.Time {
+	layout := "1/02/2006 15:04:05"
+	t:= ParseDate(layout, date)
```

Comment:
I don't think there's any benefit on declaring a variable here. This would be better as a one-liner.